src/Main.java

```diff
@@ -0,0 +1,28 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by avery on 12/6/16.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        ArrayList<Character> characters = new ArrayList();
+        characters.add(new Ranger("Ranger 1", 0.75));
+        characters.add(new Ranger("Ranger 2", 1.0));
+        characters.add(new Ranger("Ranger 3", 1.25));
+        for(int i=0; characters.size()!=1;i++){
+            if(i>=characters.size()){
+                i=0;
+            }
+            if(!characters.get(i).isAlive()){
+                characters.remove(i);
+                System.out.println("Removed");
+                continue;
+            }
+
+            characters.get(i).attack(characters.get(characters.get(i).pickTarget(characters)));
+        }
+        characters.get(0).describeSelf();
```

Comment:
Instead of having the character describe itself, just print out a happy message and the name of the character that won